packages/webvh/src/dids/didWebvhUtil.ts

```diff
@@ -0,0 +1,94 @@
+import { type AgentContext, Buffer, DidDocument, JsonTransformer, KeyType, Key } from '@credo-ts/core'
+import { createDID, AbstractCrypto, type SigningOutput, type SigningInput } from 'didwebvh-ts'
+
+export class DIDWebvhCrypto extends AbstractCrypto {
+  private agentContext?: AgentContext
+
+  public constructor(agentContext?: AgentContext) {
+    super({
+      verificationMethod: {
+        id: 'did:webvh:123',
+        controller: 'did:webvh:123',
+        type: 'Ed25519VerificationKey2020',
+        publicKeyMultibase: '123',
+        secretKeyMultibase: '123',
+      },
+    })
+    this.agentContext = agentContext
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  public async sign(input: SigningInput): Promise<SigningOutput> {
+    throw new Error('Not implemented')
+  }
+
+  public async verify(signature: Uint8Array, message: Uint8Array, publicKey: Uint8Array): Promise<boolean> {
+    try {
+      if (!this.agentContext) {
+        throw new Error('Agent context is required')
+      }
+
+      const key = new Key(publicKey, KeyType.Ed25519)
+
+      return await this.agentContext.wallet.verify({
+        key,
+        data: Buffer.from(message),
+        signature: Buffer.from(signature),
+      })
+    } catch (error) {
+      // Log error in a non-production environment
```

Comment:
You can use agentContext.config.logger.error for this, and can be disabled by the user of the framework